src/rgw/driver/sfs/sqlite/dbconn.h

```diff
@@ -292,12 +294,20 @@ class DBConn {
   static std::string getDBPath(CephContext* cct) {
     auto rgw_sfs_path = cct->_conf.get_val<std::string>("rgw_sfs_data_path");
     auto db_path =
-        std::filesystem::path(rgw_sfs_path) / std::string(SCHEMA_DB_NAME);
+        std::filesystem::path(rgw_sfs_path) / std::string(DB_FILENAME);
+    return db_path.string();
+  }
+
+  static std::string getLegacyDBPath(CephContext* cct) {
```

Comment:
to be perfectly honest, I really dislike that we have this function as camelCase when all other functions are snake case. More of an itch for me than anything else.